web/template/team-remove-member.gotmpl

```diff
@@ -0,0 +1,50 @@
+{{ template "page" . }}
+
+{{ define "backlink" }}
+  <a class="govuk-back-link" href="{{ prefix (printf "/teams/%d" .Team.ID) }}">Back</a>
+{{ end }}
+
+{{ define "title" }}Remove users from team{{ end }}
+
+{{ define "main" }}
+  {{ template "error-summary" .Errors }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Remove users from team</h1>
+
+      {{ if gt (len .Selected) 2 }}
+        <p class="govuk-body">
+          Are you sure you want to remove the following members from the <strong>{{ .Team.DisplayName }}</strong> team?
```

Comment:
I think this is clearer even for the smaller cases, but don't really mind either way

Reply:
I've switched to be a one-liner for individuals and a list for any multiples